Cabal-tests/tests/ParserTests/regressions/public-multilib-2.cabal

```diff
@@ -9,6 +9,6 @@ library
   default-language: Haskell2010
   build-depends:
     , base     ^>=4.14
-    , somelib:internal
+    , somelib:internal ^>=1.0
```

Comment:
I'm uncertain what this syntax `somelib:internal` means, I assumed `somelib` is a dependency.
There appears no docs on this syntax (note that this is the other multilib failing now, not mulitlib 1, which I fixed).

Reply:
I think somelib is a dep, internal is its component.